runtime/nvqir/qpp/QppCircuitSimulator.cpp

```diff
@@ -232,45 +207,52 @@ class QppCircuitSimulator : public nvqir::CircuitSimulatorBase<double> {
 
   /// @brief Reset the qubit
   /// @param qubitIdx
-  void resetQubit(const std::size_t qubitIdx) override {
+  void resetQubit(const std::size_t index) override {
```

Comment:
Recommend updating `@param` with new parameter name and providing a valid description of the parameter (i.e. `@param index 0-based index of qubit to reset`).